chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -2641,14 +2641,16 @@ async def test_check_removals_with_block_creation(flags: int) -> None:
 
 @pytest.mark.anyio
 async def test_dedup_not_canonical() -> None:
-    # this is 1, but with a non-canonical encoding
-    coin_spend = mk_coin_spend(TEST_COIN, solution="c00101")
-    coins = TestCoins(coins=[], lineage={})
+    # this is ((1)), but with a non-canonical encoding
+    coin_spend = mk_coin_spend(TEST_COIN, solution="ffffc001018080")
+    coins = TestCoins([TEST_COIN], lineage={})
     mempool_manager = await setup_mempool(coins)
     sb = SpendBundle([coin_spend], G2Element())
     sb_conds = make_test_conds(spend_ids=[(TEST_COIN, ELIGIBLE_FOR_DEDUP)])
     bundle_add_info = await mempool_manager.add_spend_bundle(sb, sb_conds, sb.name(), uint32(1))
+    print(bundle_add_info)
```

Comment:
Intentionally left here?

Reply:
yes

Reply:
I read *unintentionally* :). So, I removed it